modnet/models/ensemble.py

```diff
@@ -89,18 +92,24 @@ def fit(
 
         if self.bootstrap:
             LOG.info("Generating bootstrap data...")
+            if self.random_state is None:
+                random_state = self.n_models * [None]
+            else:
+                random_state = np.arange(self.n_models) + self.random_state
+
+            train_indices = [
+                resample(
```

Comment:
Hi @ml-evs could you please perform the resample with stratify in case of classification tasks? This would be very helpful(:

Reply:
In principle, I don't see why not, but I'm not sure how this interacts with the joint learning of classification/regression tasks @ppdebreuck -- I guess we have to loop through all properties and see if any are classficiation, and if so, stratify? I'll push a commit doing this, but the whole switch around `num_classes` needs to be rewritten IMO

Reply:
I've pushed something naive in 9e2f22de2f227aa800ef84345a27db621fca1378, its not clear if this will work seamlessly with dataframes and/or joint-learning classification or not...